package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -822,94 +1109,208 @@ def timeseries(self):
            w.run()
            timeseries = w.timeseries
 
-        See Also
-        --------
-        :attr:`table` : structured array of the data
         '''
-        return super(WaterBridgeAnalysis, self).timeseries
-
-    def generate_table(self):
-        """Generate a normalised table of the results.
 
-        The table is stored as a :class:`numpy.recarray` in the
-        attribute :attr:`~WaterBridgeAnalysis.table`.
-
-        See Also
-        --------
-        WaterBridgeAnalysis.table
-        """
-        super(WaterBridgeAnalysis, self).generate_table()
-
-    def count_by_type(self):
+        def analysis(current, output):
+            output.extend(current)
+
+        timeseries = []
+        for frame in self._network:
+            new_frame = []
+            self._traverse_water_network(frame, [], analysis_func=analysis, output=new_frame, link_func=self._full_link)
+            timeseries.append(new_frame)
+        self._timeseries = timeseries
+        # after 1.0 will be return the unformated _timeseries
```

Comment:
If you're making other breaking changes then you might as well get rid of `_reformat_hb` now and not wait until 1.0

Reply:
@orbeckst I'm just copying this from Hydrogen bond analysis and this can surely be removed.